deployment/confStorage/download.py

```diff
@@ -73,6 +73,11 @@ def download_cluster_configuration(self, local_path):
             self.logger.error("The configuration doesn't exists on your cluster. Please upload it first.")
             sys.exit(1)
 
+        if u'config.yaml' not in configuration_dict:
+            self.logger.warning("config.yaml doesn't exists on your cluster.")
+            self.logger.warning("Before v1.7.0, config.yaml is stored in ~/pai-deploy/cluster-cfg/config.yaml on the dev box machine.")
```

Comment:
Suggest adding a link for `config.yaml` schema reference, in case the admin user lose the backup